### ISSUE TITLE
Windows does not support the emojis/graphics

### ISSUE THREAD

**Issue by AlexStormwood** [BlueCodeThree/Ruby-Snake#1]
It's playable, but the graphics don't match the epic stuff I've seen!

EDIT: Windows running the WSL / Ubuntu for Windows 10 system.

![WindowsRubySnake](https://user-images.githubusercontent.com/22165407/56171544-0e383100-6029-11e9-9073-80316e34f3d5.PNG)

**Comment by CarlieHamilton** [comment]
Yes I know! I have been thinking about this. It works in VS code terminal (wsl for windows 10 here). Will have to see what I can do, if there is a gem (probably?), or to just have a unicode option. *thinking emoji*
